### ISSUE TITLE
ESLint migrator should ignore disabled unsupported rules

### ISSUE THREAD

**Issue by fireairforce** [utooland/utoo-lint#1058]
## What to build

When `utoo-lint migrate eslint` encounters a rule that is not implemented by utoo-lint but is disabled in the ESLint config, omit it from the blocking `unsupportedRules` report.

Disabled rules have no runtime behavior. Shared configs and formatter compatibility configs often contain many disabled plugin rules, so treating them as unsupported makes an otherwise usable migration exit with status 1.

Minimal reproduction:

```js
export default [
  {
    rules: {
      "example/not-implemented": "off",
      "no-debugger": "error",
    },
  },
];
```

With `@utoo/lint@0.2.8`, migrating this config produces a valid `no-debugger` rule but still reports `example/not-implemented` as unsupported and exits with status 1.

## Acceptance criteria

- [ ] Unsupported rules configured as `"off"`, `0`, `false`, or an array whose severity is disabled do not make migration fail.
- [ ] Disabled unsupported rules are either omitted from the report or listed separately as non-blocking information.
- [ ] Enabled unsupported rules continue to be reported and keep the non-zero exit status.
- [ ] Tests cover scalar and array severity forms.

## Blocked by

None - can start immediately
